order/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db      import models
+from user.models    import Users
+from product.models import Products
+
+class Payments(models.Model):
+    user       = models.models.ForeignKey(Users, on_delete=models.CASCADE)
+    card       = models.CharField(max_length=50)
+    createtime = models.DateField(auto_now_add=True)
```

Comment:
하나의 결제에는 하나의 주문만 가능하도록 구현해주세요~
```suggestion
    order       = models.OneToOneField(Orders, on_delete=models.CASCADE)
    card       = models.CharField(max_length=50)
    amount = ..
    payment_method = ..
    created_at = models.DateField(auto_now_add=True)
    updated_at = ..
```

Reply:
모델 수정을 하고 있는데 amout에 필요성을 찾지 못했습니다. 그 이유는 amount가 양을 측정하는 column인데  양을 측정하는 부분은 Carts에 count를 column으로 설정하였습니다